doc/rest_api/snippet_sharing/api-description.apib

```diff
@@ -0,0 +1,587 @@
+FORMAT: 1.0
+
+# REST service API for sharing configuration snippets
+
+This is a description of the API provided by the REST service that can be used to share, search and download configuration snippets.
+
+
+# API Description [/]
+
+## List API methods [GET]
+
+Returns an overview of available API methods with required parameters and a description.
+
++ Response 200 (text/html; charset=utf-8)
+
+
+## API Version [GET /version]
+
+Returns the currently used version of the API. A version consists of two numbers (format: a.b), where the first number (a) is the major version and the second number (b) the minor version. Minor versions may not contain breaking changes (i.e. backwards-compatibility is ensured), but major versions may. This means that minor versions may only extend the API.
```

Comment:
There are 3 numbers, see doc/VERSION.md. I do not think you need to repeat the explanation of what the numbers mean, can you link to the file?